perf/impl/go-libp2p/v0.28/.gitignore

```diff
@@ -1,3 +1,2 @@
 perf
 .cache
-v0.28
```

Comment:
I couldn't find anywhere where this would get created.

Reply:
This is created when one runs `go build .`. That said, fine to remove it.